src/pages/Script/ScriptsComp.tsx

```diff
@@ -122,33 +130,46 @@ export const ScriptCells = ({
 }) => {
   const history = useHistory()
   const { codeHash, hashType } = useParams<{ codeHash: string; hashType: string }>()
-
-  const cellsQuery = useQuery([`scripts_${cellType}`, codeHash, hashType, page, size], async () => {
-    const { data, meta } = await explorerService.api.requesterV2
-      .get(`scripts/${cellType}`, {
-        params: {
-          code_hash: codeHash,
-          hash_type: hashType,
-          page,
-          page_size: size,
-        },
-      })
-      .then((res: AxiosResponse) =>
-        toCamelcase<Response.Response<{ deployedCells?: CellInScript[]; referringCells?: CellInScript[] }>>(res.data),
-      )
-    const camelCellType = camelcase(cellType) as 'deployedCells' | 'referringCells'
-    if (data == null) {
-      throw new Error('Fetch Cells null')
-    }
-    const cells = data[camelCellType]!
-    if (cells == null || cells.length === 0) {
-      throw new Error('Cells empty')
-    }
-    return {
-      total: meta?.total ?? 0,
-      cells,
-    }
+  const [cellsEmpty, setCellsEmpty] = useState<Record<'deployedCells' | 'referringCells', boolean>>({
+    deployedCells: false,
+    referringCells: false,
   })
+
+  const camelCellType = camelcase(cellType) as 'deployedCells' | 'referringCells'
```

Comment:
I think the type constraint of the incoming `cellType` should be directly changed to `'deployedCells' | 'referringCells'`, and then use different api addresses according to `cellType` when requesting, similar to `get(scripts/${cellType === 'deployedCells' ? 'deployed_cells' : ...}` .